docs/charts/line.md

```diff
@@ -172,34 +178,14 @@ The line chart defines the following configuration options. These options are me
 It is common to want to apply a configuration setting to all created line charts. The global line chart settings are stored in `Chart.defaults.line`. Changing the global options only affects charts created after the change. Existing charts are not changed.
 
 For example, to configure all line charts with `spanGaps = true` you would do:
+
 ```javascript
 Chart.defaults.line.spanGaps = true;
 ```
 
 ## Data Structure
 
-The `data` property of a dataset for a line chart can be passed in two formats.
-
-### number[]
-```javascript
-data: [20, 10]
-```
-
-When the `data` array is an array of numbers, the x axis is generally a [category](../axes/cartesian/category.md#category-cartesian-axis). The points are placed onto the axis using their position in the array. When a line chart is created with a category axis, the `labels` property of the data object must be specified.
-
-### Point[]
-
-```javascript
-data: [{
-    x: 10,
-    y: 20
-}, {
-    x: 15,
-    y: 10
-}]
-```
-
-This alternate is used for sparse datasets, such as those in [scatter charts](./scatter.md#scatter-chart). Each data point is specified using an object containing `x` and `y` properties.
+See [`Data structures`](../general/data-structures)
```

Comment:
Does this work? I've normally seen the other links include `.md` at the end

Reply:
Probably not - not sure where I got that. Changed